.github/CODEOWNERS

```diff
@@ -23,6 +23,7 @@
 /pytorch_lightning/callbacks            @williamfalcon @tchaton @carmocca @borda @kaushikb11
 /pytorch_lightning/core                 @tchaton @SeanNaren @borda @carmocca @justusschock @kaushikb11
 /pytorch_lightning/distributed          @williamfalcon @tchaton @awaelchli @kaushikb11
+/pytorch_lightning/lite                 @tchaton @awaelchli @carmocca
```

Comment:
These 3 are the minimum because they have made strong contributions in form of engineering, discussion, design, and documentation.